Car_owners/tests/test_models.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from application.models import Owner, Car
+
+
+@pytest.fixture
+def model_data() -> dict[str, Owner | Car]:
+    owner = Owner.objects.create(name="Adam", surname="Knafel", phone="123456789")
+    car = Car.objects.create(
+        brand="Ford", model="Focus", production_date="2023-01-01", owner=owner
+    )
+
+    return {"owner": owner, "car": car}
+
+
+@pytest.mark.django_db
+def test_owner_display_str_method(model_data: dict[str, Owner | Car]) -> None:
+    assert (
+        model_data["owner"].__str__()
```

Comment:
this test doesn't present much business value since you're just copying what you've specified in the model's method definition, but you can keep it for coverage if you want. I would suggest using `str()`, private methods aren't meant to be called from outside the class